src/client/resources/convension.ts

```diff
@@ -0,0 +1,22 @@
+export type INamespaceList = string[];
```

Comment:
I'm not sure about where this lives as I mentioned, let's talk through it.

Reply:
Really it is just a list of strings, let's remove this.